openbb_platform/core/openbb_core/provider/standard_models/government_trades.py

```diff
@@ -0,0 +1,45 @@
+"""Government Trades Standard Model."""
+
+from datetime import date as dateType
+from typing import Literal, Optional
+
+from openbb_core.provider.abstract.data import Data
+from openbb_core.provider.abstract.query_params import QueryParams
+from openbb_core.provider.utils.descriptions import (
+    DATA_DESCRIPTIONS,
+    QUERY_DESCRIPTIONS,
+)
+from pydantic import Field, NonNegativeInt, field_validator
+
+
+class GovernmentTradesQueryParams(QueryParams):
+    """Government Trades Query."""
+
+    symbol: Optional[str] = Field(
+        default=None, description=QUERY_DESCRIPTIONS.get("symbol", "")
+    )
+    chamber: Literal["house", "senate", "all"] = Field(
+        default="all", description="Government Chamber."
+    )
+    limit: Optional[NonNegativeInt] = Field(
+        default=100, description=QUERY_DESCRIPTIONS.get("limit", "")
+    )
+
+    @field_validator("symbol", mode="before", check_fields=False)
+    @classmethod
+    def to_upper(cls, v: str) -> str:
+        """Convert field to uppercase."""
+        return v.upper() if v else None
+
+
+class GovernmentTradesData(Data):
+    """Government Trades data."""
+
+    symbol: Optional[str] = Field(description=DATA_DESCRIPTIONS.get("symbol", ""))
```

Comment:
This needs to have a default value of None, otherwise Pydantic it's still mandatory.

```python
obb.equity.ownership.government_trades()

OpenBBError: 
[Error] -> 1 validations error(s)
[Data Model] FMPGovernmentTradesData
[Arg] symbol -> input: {'date': '2024-12-03', 'transaction_date': '2024-12-02', 'assetDescription': 'Treasury Bill', 'type': 'Sale (Full)', 'amount': '$15,001 - $50,000', 'representative': 'Rudy Yakym', 'link': 'https://disclosures-clerk.house.gov/public_disc/ptr-pdfs/2024/20026311.pdf'} -> Field required
```

Reply:
Got it, I’ve fixed it now.